### ISSUE TITLE
Modify unity-app-generator to create Dockstore.cwl that calls main workflow CWL

### ISSUE THREAD

**Issue by mcduffie** [unity-sds/unity-app-generator#8]
The Dockstore API requires a default CWL workflow called Dockstore.cwl. Automate the unity-app-generator process to create this file automatically and have it call the process.cwl file created by app-pack-generator.

**Comment by mcduffie** [comment]
Example file:

```
cwlVersion: v1.0

class: Workflow
doc: |
    Bogus hosted workflow registered by unity_py.

steps:
step:
    run: step.cwl

s:author:
  - class: s:Person
    s:email: mliukis@jpl.nasa.gov
    s:name: Masha Liukis
```

**Comment by mcduffie** [comment]
Implemented at 5d60ece1ebbc25801982bad1d635418c28905397
